Header header

int32 CMD_NECK                = -2
int32 CMD_SAVE_AND_QUIT       = -1
int32 CMD_IDLE                = 0
int32 CMD_WALK                = 1
int32 CMD_GESTURE_START       = 2
# 3 and higher are gestures

int32 cmd
float64[] param  
